core/storage.js

```diff
@@ -52,17 +26,17 @@ define('core/storage',
             }
         },
         removeItem: function(key) {
-            ls.removeItem(mapKey(key));
+            localStorage.removeItem(mapKey(key));
         },
         setItem: function(key, value) {
             try {
-                ls.setItem(mapKey(key), JSON.stringify(value));
+                localStorage.setItem(mapKey(key), JSON.stringify(value));
             } catch (e) {
                 // Clear localStorage if the quota was reached.
                 if (e.name == 'QuotaExceededError' ||
                     e.name == 'NS_ERROR_DOM_QUOTA_REACHED') {
-                    console.log('LocalStorage full, clearing and reloading');
-                    ls.clear();
+                    require('core/log')('storage').log('LS full, clearing');
```

Comment:
the tracking idea is good. the user token part is a bit messy since we expect to have a lot of other stuff like user's apps, permissions, settings.